WRN-5603: Fixed `FormCheckboxItem`'s checkbox color in disabled focused state (#1112)
enactjs/sandstone: CHANGELOG.md

- Support color customization

### Fixed

> - `sandstone/FormCheckboxItem` to show correct color for a checkbox in disabled and focused state when `slotBefore` is given

Reviewer: ```suggestion
- `sandstone/FormCheckboxItem` to show correct color for the focused disabled checkbox
```
